charts/tensorleap/charts/node-server/templates/mongodb-deploy.yaml

```
@@ -16,6 +16,8 @@ spec:
     metadata:
       labels:
         app: mongodb
+      annotations:
+        ad.datadoghq.com/mongodb.logs: '[{"source": "mongo", "service": "node-server"}]'
```

```suggestion
        ad.datadoghq.com/mongodb.logs: '[{"source": "mongo", "service": "mongo"}]'
```
So we can split between the logs more easily in ddog